lua/telescope/actions/init.lua

```diff
@@ -299,6 +299,23 @@ actions.select_tab = {
   end,
 }
 
+--- Perform 'tab drop' action on selection, usually something like<br>
+---`:tab drop <selection>`
+---
+--- i.e. open the selection in a new tab
+---@param prompt_bufnr number: The prompt bufnr
+actions.select_tab_drop = {
```

Comment:
do we also want a `select_drop`? and we forgot to add it in the previous PR

(same comment as above)

Reply:
https://github.com/rkoval/telescope.nvim/commit/5cc4bf49fe72e583e2529bdf7b6e0f108c07dcbf